kahuna/conf/routes

```diff
@@ -17,6 +17,8 @@ GET     /login                      controllers.Panda.doLogin(redirectUri: Optio
 GET     /oauthCallback              controllers.Panda.oauthCallback
 GET     /logout                     controllers.Panda.logout
 GET     /session                    controllers.Panda.session
+# Empty page to return to the same domain as the app
+GET     /ok                         controllers.Application.ok
```

Comment:
I would rather not.

Healthcheck is for healthcheck purposes (ELB, Pingdom, other monitoring tools). It's a system-facing endpoint.

The "OK" page is a user-facing landing page.